libs/sdk-ui-kit/src/Header/HeaderUpsellButton.tsx

```diff
@@ -0,0 +1,21 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+import { FormattedMessage, injectIntl, IntlShape } from "react-intl";
+
+export interface IUpsellButtonProps {
+    intl: IntlShape;
+    onUpsellButtonClick?: (e: React.MouseEvent) => void;
+}
+
+export const CoreHeaderUpsellButton: React.FC<IUpsellButtonProps> = ({ onUpsellButtonClick }) => {
+    return (
+        <button className="gd-button-small gd-button-primary gd-upsell-button" onClick={onUpsellButtonClick}>
+            <i className="gd-icon-star" />
+            <span className="gd-upgrade-button-text">
+                <FormattedMessage id="gs.header.upsellButtonText" />
```

Comment:
Should we add the English translation for `gs.header.upsellButtonText` in [en-US.json](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui/src/base/localization/bundles/en-US.json) so that we don't have to care about adding the translations for it in each of the apps.

Reply:
I added the translation thanks